opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableCreation.jsx

```diff
@@ -227,110 +233,279 @@ const StixCyberObservableCreation = ({
   const { isVocabularyField, fieldToCategory } = useVocabularyCategory();
   const { booleanAttributes, dateAttributes, multipleAttributes, numberAttributes, ignoredAttributes } = useAttributes();
   const [status, setStatus] = useState({ open: false, type: type ?? null });
-
+  const [openProgressDialog, setOpenProgressDialog] = useState(false);
   const handleOpen = () => setStatus({ open: true, type: status.type });
   const localHandleClose = () => setStatus({ open: false, type: type ?? null });
   const selectType = (selected) => setStatus({ open: status.open, type: selected });
+  const [genericValueFieldDisabled, setGenericValueFieldDisabled] = useState(false);
+  const bulkAddMsg = t_i18n('Multiple values entered. Edit with the TT button');
```

Comment:
Should be aligned with the new button

Reply:
Completed in latest commit 9199da1c3604c14e2ddbf1bcfd6f37b9e5c4a4e6

Reply:
Translations added in commit acbca1f559f530dd6b627c4aded241758c9002f8